src/StreetNameRegistry.Consumer/Projections/MunicipalityKafkaProjection.cs

```diff
@@ -234,13 +235,13 @@ public static IHasCommandProvenance GetCommand(IQueueMessage message)
             throw new InvalidOperationException($"No command found for {type.FullName}");
         }
 
-        private static Provenance FromProvenance(Contracts.Provenance provenance) =>
+        private static Provenance FromProvenance(Contracts.Provenance provenance, Modification modification = Modification.Update) =>
```

Comment:
I think in all other cases besides when importing a municipality, it should be Update. Is my thinking correct?

Reply:
Yes